test/utils/ims/ims.test.html

```diff
@@ -0,0 +1,181 @@
+<html>
+  <head>
+    <!-- the import map to use in our test -->
+    <script type="importmap">
+      {
+        "imports": {
+          "../../../nx/scripts/nexter.js": "./mocks/mock-nexter.js"
```

Comment:
This is also extremely clever. Can we import nexter.js and then override the function we need and then export? Thinking we don't have to mock the whole thing, but I probably need to pull this down to see.